documentation/src/main/asciidoc/topics/con_xsite_concurrent_writes.adoc

```diff
@@ -56,6 +60,31 @@ Prepend site names with numbers as a simple way to represent the order of
 priority for resolving conflicting entries; for example, **1LON** and **2NYC**.
 ====
 
+== Custom Conflict Resolution
```

Comment:
I think using "Custom" here to refer to all merge policies is confusing, because we usually write "custom" when users are implementing an interface themselves.